Kitodo/src/main/java/org/kitodo/services/file/FileService.java

```diff
@@ -191,7 +223,8 @@ public URI renameFile(URI fileUri, String newFileName) throws IOException {
                 }
                 millisWaited += SLEEP_INTERVAL_MILLIS;
             }
-        } while (!success && millisWaited < MAX_WAIT_MILLIS);
+        }
```

Comment:
actually it is interesting case, auto formatter says:
` } while`
and checkstyle says
```
} 
while
```